cl/citations/match_citations.py

```diff
@@ -139,3 +146,110 @@ def match_citation(citation, citing_doc=None):
 
     # Give up.
     return []
+
+
+def get_citation_matches(opinion, citations):
+    # A list of opinions, as matched to citations
```

Comment:
Can you add a proper docstring here please?

Reply:
👍

Reply:
Fixed in c839569